compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Repl.hs

```diff
@@ -325,8 +328,8 @@ runRepl importPkgs opts replClient logger ideState = do
           where
             banner = pure "daml> "
             command = replLine
-            options = []
-            prefix = Nothing
+            options = replOptions
```

Comment:
Just for my own clarification: Do you know why `replline` special cases commands? Is there any advantage to using this over implementing it in the `replLine` function?

Reply:
It automatically generates short-cuts like `:m` for `:modules` and generates a "no such command" error in case of unknown commands.

Reply:
Thanks for the explanation!